dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -19,10 +20,9 @@ def show_diversity_survey?(kind)
   end
 
   def show_nps_survey?
-    return false unless SurveyResultsHelper::NPS_SURVEY_ENABLED
+    return false unless NPS_AUDIENCE != "none"
```

Comment:
It might also be ok to remove this line entirely (since this logic is covered by target_audience?, right?).

Reply:
good point- removed:)